internal/code-generator/generator/drop_whitespace_test.go

```diff
@@ -0,0 +1,29 @@
+package generator
+
+import (
+	"strings"
+	"testing"
+)
+
+var wdata = " eityh edsfsdf    dsfds			dsfsdf 	\t\t\t dfsdfsdf sdfsddf"
+
+func dropSpacesJoin(str string) string {
+	return strings.Join(strings.Fields(str), "")
```

Comment:
I recommend normalizing spaces rather than dropping them. Otherwise `"a bc d"` looks equivalent to `"ab cd"` even though they are different token streams.